### PR TITLE
wcag: update button color when focus/hover

### PR DESCRIPTION
**Pull request by margaree**
Improve the contrast ratio for button text when button is hovered/focused.

`headless` was removed from running tests locally because of an issue with browser window focus causing `.focus()` to not run when `headless` and for the tests with `.focus()` to timeout. A travis wct config was added to run the tests headlessly, since they fail on travis when not run with `headless`.

I tried to use `MockInteractions.focus` here instead of `.focus()` but that seemed to not actually trigger the styles.

### CODE REVIEW

File: wct.conf.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "local": {
       "browsers": ["chrome"],
       "browserOptions": {
-        "chrome": ["headless", "disable-gpu", "no-sandbox"]
+        "chrome": ["disable-gpu", "no-sandbox"]
```
Comment (dlockhart) [inline wct.conf.json]: Curious about whether there's a performance penalty we pay for not running headless. I guess if it's just for the local tests it's not a big deal. If it ends up getting costly we could always split our tests into 2 sets -- one that runs headlessly and one that doesn't.

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "test:diff": "mocha './test/**/*.visual-diff.js' -t 10000",
     "test:diff:golden": "mocha './test/**/*.visual-diff.js' -t 10000 --golden",
     "test:local": "polymer test --skip-plugin sauce",
-    "test:sauce": "polymer test --skip-plugin local"
+    "test:sauce": "polymer test --skip-plugin local",
+    "test:travis": "polymer test --config-file wct.conf-travis.json"
```
Comment (dlockhart) [inline package.json]: I think `--skip-plugin` can take a list of plugins... so in theory if you didn't want to introduce the config file you could create a 3rd plugin in the `wct.conf.json` file just for travis.

File: wct.conf.json (hunk 1 above)
Comment (margaree) [reply wct.conf.json]: It's probably a bit slower but it runs in a few seconds right now. Agreed about splitting should we need to.

File: test/button/button-subtle.html [hunk 3]
```diff
@@ -55,8 +55,7 @@
 						await runAxe(button);
 					});
 
-					// TODO: Fix contrast of focussed buttons to meet WCAG 2 standards
```
Comment (svanherk) [inline test/button/button-subtle.html]: 🎉

File: package.json (hunk 2 above)
Comment (margaree) [reply package.json]: I think the plugins are more "official" than that - I get an error `Could not find WCT plugin named "travis"`. Looking into if there's a way around that.

Comment (margaree) [reply package.json]: Yeah there doesn't seem to be a way to get custom plugin names that I have found

Comment (dlockhart) [reply package.json]: OK, thanks for trying!

**Pull request merged**